config/management/operational/src/command.rs

```diff
@@ -243,6 +254,16 @@ impl Command {
         }))
     }
 
+    /// Show VerifyValidatorState result
+    fn print_verify_validator_state_result(
```

Comment:
Currently I'm implementing a simple `print` for displaying `VerifyValidatorStateResult`. We can iterate on it later. One way I'm thinking of is, displaying overall result (i.e., state being consistent or not), then display result for each individual check  after that.